src/SDKs/PolicyInsights/Management/PolicyInsights.Tests/SessionRecords/PolicyInsights.Tests.PolicyTrackedResourcesTests/PolicyTrackedResourcesManagementGroupScopeQueries.json

```diff
@@ -0,0 +1,2120 @@
+{
```

Comment:
The filenames for the sessionRecords don't look right.  They shoudl match the test name exactly I believe (they do for the existing session records).  Do these work in playback mode?

Reply:
They do match the session names, I just need to fix the MG test name.